core/src/main/java/org/apache/servicecomb/core/handler/HandlerConfigUtils.java

```diff
@@ -31,8 +31,9 @@ private HandlerConfigUtils() {
   private static Config loadConfig() throws Exception {
     Config config = new Config();
 
-    List<Resource> resList =
-        PaaSResourceUtils.getSortedResources("classpath*:config/cse.handler.xml", ".handler.xml");
+    List<Resource> resList = PaaSResourceUtils.getResources(new String[]{"classpath*:config/*.handler.xml"});
+    PaaSResourceUtils.sortResources(resList, ".handler.xml");
```

Comment:
Suffix is not correct. Add test cases to make sure file name sorting is work as expected.

Reply:
Format code